step2-05/README.md

```diff
@@ -12,17 +12,19 @@ f(data) => view
 
 This is fine if the data never changes. However, React exists to deal with data updates via props (immutable) and state (changes based on `setState()` API). In the real world, data is shaped like a tree and view is shaped like a tree. They don't always match.
 
+We can pass shared state data down to components using props and update it using callbacks, but as an application grows larger, passing around callbacks becomes unwieldy and the state data flow becomes hard to understand. To keep the application maintainable, we need a well-defined pattern for sharing and updating application state.
```

Comment:
Added more on the motivation for Redux (as I understand it--let me know if this is right!)